Doc/library/ast.rst

```diff
@@ -2472,6 +2472,19 @@ effects on the compilation of a program:
    .. versionadded:: 3.8
 
 
+.. function:: compare(a, b, /, *, compare_attributes=False)
+
+   Recursively compares two ASTs.
+
+   *compare_attributes* affects whether AST attributes are considered
+   in the comparison. If compare_attributes is ``False`` (default), then
+   attributes are ignored. Otherwise they must all be equal. This
+   option is useful to check whether the ASTs are structurally equal but
+   might differ in whitespace or similar details.
```

Comment:
We could be more explicit in saying that "attributes" includes the line and column numbers.